From b28ea989d67366a6956ec6fc80dce423362511d4 Mon Sep 17 00:00:00 2001
From: Jianqun Xu <jay.xu@rock-chips.com>
Date: Thu, 25 Aug 2016 14:09:36 +0800
Subject: [PATCH] ARM64: dts: rk3399-evb2: ajust cpu opp table to make evb2
 more stable

Since evb2 couple with ES1, the power consumption is large enough to
shutdown device in some case.

Let's reduce it's support lists to make things simple.

Change-Id: I145aa0c6a21e41b3c8e6ff32fd15839baa15f81e
Signed-off-by: Jianqun Xu <jay.xu@rock-chips.com>
---
 .../boot/dts/rockchip/rk3399-evb-rev2.dtsi    | 38 +++++++++++--------
 1 file changed, 22 insertions(+), 16 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-evb-rev2.dtsi b/arch/arm64/boot/dts/rockchip/rk3399-evb-rev2.dtsi
index 941d75ac6f3e..b64a126a11e2 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-evb-rev2.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399-evb-rev2.dtsi
@@ -96,14 +96,17 @@
 	opp@1200000000 {
 		opp-hz = /bits/ 64 <1200000000>;
 		opp-microvolt = <925000>;
+		status = "disabeld";
 	};
 	opp@1416000000 {
 		opp-hz = /bits/ 64 <1416000000>;
-		opp-microvolt = <1025000>;
+		opp-microvolt = <1050000>;
+		status = "disabeld";
 	};
 	opp@1512000000 {
 		opp-hz = /bits/ 64 <1512000000>;
 		opp-microvolt = <1075000>;
+		status = "disabeld";
 	};
 };
 
@@ -119,23 +122,26 @@
 	};
 	opp@816000000 {
 		opp-hz = /bits/ 64 <816000000>;
-		opp-microvolt = <800000>;
+		opp-microvolt = <825000>;
 	};
 	opp@1008000000 {
 		opp-hz = /bits/ 64 <1008000000>;
-		opp-microvolt = <850000>;
+		opp-microvolt = <875000>;
 	};
 	opp@1200000000 {
 		opp-hz = /bits/ 64 <1200000000>;
-		opp-microvolt = <925000>;
+		opp-microvolt = <950000>;
+		status = "disabeld";
 	};
 	opp@1416000000 {
 		opp-hz = /bits/ 64 <1416000000>;
 		opp-microvolt = <1025000>;
+		status = "disabeld";
 	};
 	opp@1608000000 {
 		opp-hz = /bits/ 64 <1608000000>;
 		opp-microvolt = <1125000>;
+		status = "disabeld";
 	};
 	opp@1800000000 {
 		opp-hz = /bits/ 64 <1800000000>;
@@ -150,9 +156,9 @@
 		341   547	/*  600MHz */
 		464   794	/*  816MHz */
 		573   1141	/* 1008MHz */
-		683   1850	/* 1200MHz */
-		805   2499	/* 1416MHz */
-		915   2922	/* 1608MHz */
+	//	683   1850	/* 1200MHz */
+	//	805   2499	/* 1416MHz */
+	//	915   2922	/* 1608MHz */
 	//	1024  3416	/* 1800MHz */
 	>;
 	idle-cost-data = <
@@ -168,9 +174,9 @@
 		179    62	/*  600M */
 		243    90	/*  816M */
 		300    126	/* 1008M */
-		357    196	/* 1200M */
-		421    246	/* 1416M */
-		449    263	/* 1512M */
+	//	357    196	/* 1200M */
+	//	421    246	/* 1416M */
+	//	449    263	/* 1512M */
 	>;
 	idle-cost-data = <
 	      6
@@ -185,9 +191,9 @@
 		341   547	/*  600MHz */
 		464   794	/*  816MHz */
 		573   1141	/* 1008MHz */
-		683   1850	/* 1200MHz */
-		805   2499	/* 1416MHz */
-		915   2922	/* 1608MHz */
+	//	683   1850	/* 1200MHz */
+	//	805   2499	/* 1416MHz */
+	//	915   2922	/* 1608MHz */
 	//	1024  3416	/* 1800MHz */
 	>;
 	idle-cost-data = <
@@ -203,9 +209,9 @@
 		179    62	/*  600M */
 		243    90	/*  816M */
 		300    126	/* 1008M */
-		357    196	/* 1200M */
-		421    246	/* 1416M */
-		449    263	/* 1512M */
+	//	357    196	/* 1200M */
+	//	421    246	/* 1416M */
+	//	449    263	/* 1512M */
 	>;
 	idle-cost-data = <
 		56
-- 
2.35.3

